pkg/graveler/committed/diff_test.go

```diff
@@ -14,6 +14,19 @@ import (
 	"github.com/treeverse/lakefs/pkg/graveler/testutil"
 )
 
+type diffTestRange struct {
+	minKey string
+	maxKey string
+	count  int64
+}
```

Comment:
This is examined using `deep.Equal`.  That's a no-op for lowercase fields :-(
https://play.golang.org/p/c7RVJ6_lslg